[TW-1208] Public api top trending (#4918)
postmanlabs/postman-docs: src/pages/docs/getting-started/exploring-public-api-network.md

To access the Public API Network, visit [postman.com/explore](https://www.postman.com/explore) or select **Explore** in the header.

> <img alt="Postman Public API Network" src="https://assets.postman.com/postman-docs/v10/public-api-network-v10-1.jpg" width="1000px"/>

Reviewer: ```suggestion
<img alt="Postman Public API Network" src="https://assets.postman.com/postman-docs/v10/public-api-network-v10-1.jpg" />
```

There are a variety of ways to find APIs, workspaces, teams, and collections:

> * **Spotlight** highlights the latest exciting, useful, and noteworthy contributions to the Public API Network, handpicked by the experts at Postman.

Reviewer 2: ```suggestion
* **Spotlight** highlights the latest contributions to the Public API Network curated by Postman experts.
```
